frontend/src/Onboarding/OnboardingInfoScreen.tsx

```diff
@@ -352,7 +352,9 @@ class OnboardingScreenComponent extends React.Component<
           {textFieldStr.length !== 0 && !!year && !!gradYear ? (
             <Link
               to={{
-                pathname: "/home",
+                // TODO: After completed course screen is merged,
+                // change this to /home, and completed courses Next Link to /signup
+                pathname: "/signup",
```

Comment:
Since courses completed just got merged, you can probably switch this now and remove the comments